IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md

```diff
@@ -0,0 +1 @@
+Length for the circular transition at the end of the cant segment, measured from the start of the circular transition to the end of the segment.
```

Comment:
Is it really a circular transition or is it a curvature smoothing corresponding to a "circular transition change of curvature"?